.pre-commit-config.yaml

```
@@ -24,6 +24,17 @@ repos:
     -   id: ruff
         args: ["--fix", "--show-fixes"]
 
+# Jobs board validation (schema-based)
+-   repo: https://github.com/python-jsonschema/check-jsonschema
+    rev: 0.33.3
```

```suggestion
    rev: 0.36.0
```

Is there a reason for using 0.33.3 instead of the latest release available? https://github.com/python-jsonschema/check-jsonschema